.azure-pipelines/template-test-minimal-conda-env-install.yml

```diff
@@ -0,0 +1,140 @@
+# File: template-test-minimal-conda-env-install.yml
+parameters:
+  - name: environment
+    type: string
+    default: prod
+    values:
+      - prod
+      - dev
+  - name: envToRepo
```

Comment:
**question:** is there some convention already followed by other pipelines for parameter names?

camelCase, snake_case, kebab-case ?

Reply:
For what I seen, Microsoft uses `camelCase` in its documentation.

Reference: 
- [Job documentation](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml)